packages/x-date-pickers/src/internals/hooks/useField/useField.utils.ts

```diff
@@ -364,6 +392,13 @@ export const splitFormatIntoSections = <TDate>(
 
   return sections.map((section) => {
     if (section.separator !== '/') {
+      if (section.separator !== null && section.separator.includes(' ')) {
+        return {
+          ...section,
+          separator: section.separator.replace(' ', '\u2069 \u2066'),
```

Comment:
If we have several spaces, should it wrap each of them with `\u2069` and `\u2066` ? Or all of them at once ? Or only the first one (which is the current behavior)

Reply:
I did not consider this case. Maybe the more generic solution is to put one ate the beginning and one at the end of the separator

Reply:
Maybe

I think two spaces is something that can occur, especially if people use formats like `DD - MM - YYYY``